pubspec.yaml

```diff
@@ -12,7 +12,9 @@ dependencies:
   args: '>=0.13.0 <2.0.0'
   charcode: ^1.1.0
   collection: ^1.10.1
+  fixnum: '>=0.10.7'
```

Comment:
the version of fixnum in the Dart SDK is 10.5. You should set it to be 10.5 here too, unless you need 10.7 features, in which case you will need to also roll the version of fixnum in the Dart SDK.

Reply:
also note: we should use `^0.10.5` so that we don't accidentally depend on a newer version with breaking changes.